### PR TITLE
feat: Allow users to specify a metric host in config files

### PR DESCRIPTION
**Pull request by ScottyFillups**
## Context

It would be convenient if users could specify a URL to send executor data to their metric services.

## Objective

Add an example of `metricHost` usage in `default.yaml` and read an environment variable, `ECOSYSTEM_METRIC_HOST`, in `custom-environment-variables.yaml`.

## References

Merge for executor-k8s-vm `metricHost` implementation: https://github.com/screwdriver-cd/executor-k8s-vm/pull/23

### CODE REVIEW

File: config/default.yaml [hunk 1]
```diff
@@ -232,5 +232,7 @@ ecosystem:
     badges: https://img.shields.io/badge/build-{{status}}-{{color}}.svg
     # Default registry to pull build containers from. Uses Docker Hub if nothing/empty string is provided
     dockerRegistry: ""
+    # Hostname for metrics service, or pushgateway
+    metric_host: http://pushgateway.example.org:9091
```
Comment (petey) [inline config/default.yaml]: Should this be camelCase instead of snake_case?

Comment (ScottyFillups) [reply config/default.yaml]: Good point, I'll fix that.

File: config/default.yaml [hunk 2]
```diff
@@ -232,5 +232,7 @@ ecosystem:
     badges: https://img.shields.io/badge/build-{{status}}-{{color}}.svg
     # Default registry to pull build containers from. Uses Docker Hub if nothing/empty string is provided
     dockerRegistry: ""
+    # Pushgateway URL for Prometheus
+    pushgatewayUrl: http://pushgateway.example.org:9091
```
Comment (minz1027) [inline config/default.yaml]: Can we comment this out? If we don't set the ENV, the executor will try to use this.

**Comment by ScottyFillups** [comment]
Changes moved to https://github.com/screwdriver-cd/queue-worker/pull/43

**Pull request closed without merging**